demisto_sdk/commands/prepare_content/prepare_upload_manager.py

```diff
@@ -2,7 +2,12 @@
 from pathlib import Path
 from typing import Optional, Union
 
-from demisto_sdk.commands.common.constants import MarketplaceVersions
+from demisto_sdk.commands.common.constants import (
+    JSON_INDENT_CONSTANT,
+    YAML_INDENT_CONSTANT,
```

Comment:
please explain to me later why they is relevant.

Reply:
because every time I write json/yml I need to redefine `indent` I created constants for this